setup.cfg

```diff
@@ -19,10 +19,6 @@ long_description_content_type = text/x-rst
 zip_safe = False
 packages = find:
 python_requires = >=3.7
-setup_requires =
-    setuptools_scm
-install_requires =
-    setuptools>=30.3.0
```

Comment:
to be clear; only this part is problematic, `setup_requires` is fine in itself, but it's still redundant with `pyproject.toml` and should not be needed at all with `python_requires = >=3.7`